modules/gitops-installing-argo-rollouts-cli-by-using-linux.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argo-rollouts-cli-on-linux_{context}"]
+= Installing Argo Rollouts CLI on Linux
+
+If you are a Linux user, you can install Argo Rollouts CLI by performing the steps described in this procedure.
+
+.Prerequisites
+
+* You have installed the {OCP} CLI (`oc`)
```

Comment:
Check out the `oc-first` common attribute.

Reply:
Fixed.